packages/terra-profile-image/src/ProfileImage.module.scss

```diff
@@ -1,4 +1,6 @@
-@import '~terra-mixins/lib/Mixins';
```

Comment:
Do we need this import for inline-svg in this file?

Reply:
Added here https://github.com/cerner/terra-core/pull/2997/commits/4067560a79a06a40a6588dcc2dd27bf85f1a2214.